### ISSUE TITLE
python3 not invoked for 'butterfly' script

### ISSUE THREAD

**Issue by jinmel** [paradoxxxzero/butterfly#91]
'butterfly help' on command line will cause error because python3 was not properly invoked.
```shell
Traceback (most recent call last):
  File "/usr/local/bin/butterfly", line 4, in <module>
    __import__('pkg_resources').run_script('butterfly==2.0.0', 'butterfly')
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 735, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 1652, in run_script
    exec(code, namespace, namespace)
  File "/usr/local/lib/python2.7/dist-packages/butterfly-2.0.0-py2.7.egg/EGG-INFO/scripts/butterfly", line 43, in <module>
    exec(compile(open(file_).read(), file_, 'exec'))
  File "/usr/local/lib/python2.7/dist-packages/butterfly-2.0.0-py2.7.egg/butterfly/bin/help.py", line 59, in <module>
    rcol=shutil.get_terminal_size()[0] - 31,
AttributeError: 'module' object has no attribute 'get_terminal_size'
```

**Comment by jinmel** [comment]
the get_terminal_size is available for python 3.3+
I tried to invoke the /usr/local/bin/butterfly with python 3.5 but then it fails at pkg_resources
how about replacing get_terminal_size with 
```python
rows, columns = subprocess.check_output(['stty', 'size']).split()
```

**Comment by jinmel** [comment]
I opened pull request on #92

**Comment by jwebcat** [comment]
+1 I am experiencing this also

**Comment by paradoxxxzero** [comment]
Merged #92
